core/src/validator.rs

```diff
@@ -662,6 +711,10 @@ impl Validator {
             config.accounts_db_test_hash_calculation,
             last_full_snapshot_slot,
         );
+        info!(
+            "Chosen backends: replaying: {:?}, banking: {:?}",
+            config.replaying_backend, config.banking_backend
+        );
```

Comment:
because cli args are hidden (https://github.com/solana-labs/solana/pull/30746/files#r1138707346), this always show default values so, no harm to starting to output for validator operators, i guess.